test/support/helpers/request-field-validation-helper.ts

```diff
@@ -0,0 +1,3 @@
+export const prepareModifiedRequest = (requestIsAnArray: boolean, modifiedRequest) => {
+  return requestIsAnArray ? [modifiedRequest] : modifiedRequest;
```

Comment:
Can we add some unit test cases please.

Reply:
A unit test will ensure if the behaviour changes for this function in future then it does not break anything.
If you create a unit test, I can then raise a PR on the back of this which run unit test cases inside `test` directory as well.

Reply:
OK,
created test, tested it locally by modifying jest config. No Jest config changes submitted.